mrmShared/linux/dkms-rpm/50-mrf.rules.in

```diff
@@ -0,0 +1,2 @@
+# UDEV rule giving access to MRF device
+KERNEL=="uio*", ATTR{name}=="mrf-pci", GROUP="@@DRV_GROUP@@", MODE="0660", RUN+="/bin/chmod g+rw %S%p/resource0 && /bin/logger UDEV rule giving access to MRF device %S%p"
```

Comment:
@jerzyjamroz , does that chmod is necessary ? My understanding is that the MODE defined should change the permission

Reply:
chmod changes other resource but there is an issue in Centos7, I leave this for eventual future improvement, mostly for the Yocto test.